timesketch/frontend-ng/src/utils/RestApiClient.js

```diff
@@ -348,19 +348,36 @@ export default {
   getSearchHistoryTree(sketchId) {
     return RestApiClient.get('/sketches/' + sketchId + /searchhistorytree/)
   },
-  // Sigma
-  getSigmaList() {
-    return RestApiClient.get('/sigma/')
+  // SigmaRule (new rules file based)
+  getSigmaRuleList() {
+    return RestApiClient.get('/sigmarule/')
   },
-  getSigmaResource(ruleUuid) {
-    return RestApiClient.get('/sigma/rule/' + ruleUuid + '/')
+  getSigmaRuleResource(ruleUuid) {
+    return RestApiClient.get('/sigmarule/' + ruleUuid + '/')
   },
-  getSigmaByText(ruleText) {
+  getSigmaRuleByText(ruleText) {
     let formData = {
       content: ruleText,
     }
-    return RestApiClient.post('/sigma/text/', formData)
+    return RestApiClient.post('/sigmarule/text/', formData)
   },
+  deleteSigmaRule(ruleUuid) {
+    return RestApiClient.delete('/sigmarule/' + ruleUuid + '/')
+  },
+  createSigmaRule(ruleText) {
+    let formData = {
+      rule_yaml: ruleText,
+    }
+    return RestApiClient.post('/sigmarule/', formData)
+  },
+  updateSigmaRule(id, ruleText) {
```

Comment:
s/ruleText/ruleYAML/

Reply:
good catch, replaced it with rule_yaml to make it consistent